### DISCUSSION TITLE
Possible ideas?

### DISCUSSION THREAD

**Discussion by Raymond-ArrA** [JaapvanEkris/openrowingmonitor#41] [Ideas]
Hi.. I am curious..

Now we use GPIO pin 17 for the wired mag or hall effect- connection.

Could it be possible to make this connection wireless, like the HR sensor? 
With use of, for example the Misuro B+ sensor or Wahoo sensor BT or ANT+ ? 

Another thing I have been wondering about.. 
Could it be possible with a dropdown in the settings, to choose the rower (from rowerProfiles.js)
Like, If it exist in that file, it appears in the dropdown, and also easier to change profile - And maybe have a restart command when changing the profile,  to automate a part of the process?

**Comment by JaapvanEkris** [comment]
> Could it be possible to make this connection wireless, like the HR sensor? With use of, for example the Misuro B+ sensor or Wahoo sensor BT or ANT+ ?

ORM is the device generating the BT and ANT+ signals, not the one recieving them. As rowing machines need a much more frequent update than a bike (as most calculations are actually dependent on acceleration/deceleration), you get a signal about once ever 5ms, where ANT+ can only handle once every 400ms, and Bluetooth once every 1000ms.

> Could it be possible with a dropdown in the settings, to choose the rower (from rowerProfiles.js) Like, If it exist in that file, it appears in the dropdown, and also easier to change profile - And maybe have a restart command when changing the profile, to automate a part of the process?

In theory, yes. But in practice you see people configure the device once, and need to adjust a lot more settings and there is no need to further adjust these settings. I think there are a lot of other things we like to do via the GUI which make a lot more difference for users (like being able to adjust interval settings).
